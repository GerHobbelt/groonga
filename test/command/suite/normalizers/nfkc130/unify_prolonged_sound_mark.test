normalize \
  'NormalizerNFKC130("unify_prolonged_sound_mark", true)' \
  "ー—―─━ｰ" \
  WITH_TYPES
