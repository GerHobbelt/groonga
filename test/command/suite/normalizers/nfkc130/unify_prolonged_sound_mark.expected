normalize   'NormalizerNFKC130("unify_prolonged_sound_mark", true)'   "ー—―─━ｰ"   WITH_TYPES
[
  [
    0,
    0.0,
    0.0
  ],
  {
    "normalized": "ーーーーーー",
    "types": [
      "katakana",
      "katakana",
      "katakana",
      "katakana",
      "katakana",
      "katakana"
    ],
    "checks": [

    ]
  }
]
